### PR TITLE
Bug/841 project question radio button

### PR DESCRIPTION
**Pull request by AllenBW**

### CODE REVIEW

File: src/client/app/components/project-form/project-form.html [hunk 1]
```diff
@@ -1,129 +1,129 @@
...
-                </div>
-            </div>
+            <input ng-click="vm.openStart($event)"
+                   class="project-field__group-input--input"
+                   name="startDate"
+                   datepicker-popup="{{vm.format}}"
+                   ng-model="vm.project.start_date"
+                   is-open="vm.openedStart"
+                   max-date="vm.project.end_date"
+                   max-date="'2015-06-22'"
```
Comment (stackus) [inline src/client/app/components/project-form/project-form.html]: Why is there a hardcoded value added here as a duplicate parameter?

File: src/client/app/components/project-question-form/project-question-form.directive.js [hunk 2]
```diff
@@ -90,12 +93,15 @@
 
       function typeChangeOk() {
         vm.projectQuestion.options.length = 0;
-        vm.projectQuestion.options.push(angular.extend({}, ProjectQuestion.optionDefaults));
-        vm.projectQuestion.options.push(angular.extend({}, ProjectQuestion.optionDefaults));
+        if (vm.projectQuestion.field_type === 'select_option') {
+          vm.projectQuestion.options.push(angular.extend({}, ProjectQuestion.optionDefaults),
+            angular.extend({}, ProjectQuestion.optionDefaults));
+        } else {
+          vm.projectQuestion.options.length = 0;
```
Comment (stackus) [inline src/client/app/components/project-question-form/project-question-form.directive.js]: Why is there an else with a command that duplicates the first line of this function?

**Comment by stackus** [comment]
Why was sorting removed from the project question answers?

**Comment by stackus** [comment]
The checkbox and radio input styling is incorrect for the UX style.

For an example of correct styling for these see the checkbox on he product create/edit form for the active input.

File: src/client/app/components/project-form/project-form.html (hunk 1 above)
Comment (AllenBW) [inline src/client/app/components/project-form/project-form.html]: No idea. Will look into it.

**Comment by AllenBW** [comment]
Didn't touch the sorting, if missing it will be readded.

**Comment by AllenBW** [comment]
You want to reuse the checkout styling for a radio button?  This produces a
noticeably condensed input, kinda looks bad.

**Comment by stackus** [comment]
For radio buttons a similarly styled circle should be used in place of the native browser element.

File: src/client/app/components/project-form/project-form.html (hunk 1 above)
Comment (AllenBW) [inline src/client/app/components/project-form/project-form.html]: addressed

File: src/client/app/components/project-question-form/project-question-form.directive.js (hunk 2 above)
Comment (AllenBW) [inline src/client/app/components/project-question-form/project-question-form.directive.js]: addressed

**Pull request merged**